tests/AppBundle/Controller/CommitteeControllerTest.php

```diff
@@ -383,7 +393,7 @@ public function testAnonymousGuestCanShowCommitteePage()
         $this->assertTrue($this->seeHosts($crawler, 2), 'The guest should see the hosts');
         $this->assertTrue($this->seeHostsContactLink($crawler, 2), 'The guest should see the hosts contact link');
         $this->assertFalse($this->seeHostNav($crawler), 'The guest should not see the host navigation');
-        $this->assertSeeSocialLinks($crawler, $this->getCommitteeRepository()->findOneByUuid(LoadAdherentData::COMMITTEE_1_UUID));
+        $this->seeSocialLinks($crawler, $this->getCommitteeRepository()->findOneByUuid(LoadAdherentData::COMMITTEE_1_UUID));
```

Comment:
Because the other new "assert" custom methods don't have this prefix. Maybe it's better to have `assert` prefix everywhere then.